components/translation/usage.rst

```diff
@@ -255,14 +255,27 @@ To translate pluralized messages, use the
 
     $translator->transChoice(
         'There is one apple|There are %count% apples',
-        10,
-        array('%count%' => 10)
```

Comment:
`Using the translator` page has a first section about this (http://symfony.com/doc/2.7/components/translation/usage.html#message-placeholders).